harvard-leeds-beckett-university.csl

```diff
@@ -0,0 +1,644 @@
+          </names>
+        </if>
+        <else-if variable="author">
+          <names variable="author">
+            <name form="short" initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-et-al="never" delimiter-precedes-last="never" and="symbol" et-al-min="3" et-al-use-first="1" />
+          </names>
+        </else-if>
+        <else-if variable="editor">
+          <names variable="editor">
+            <name form="short" initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-et-al="never" delimiter-precedes-last="never" and="symbol" et-al-min="3" et-al-use-first="1" />
+          </names>
+        </else-if>
+        <else-if type="webpage  entry-encyclopedia  entry-dictionary  post-weblog">
+          <choose>
+            <if variable="container-title">
+              <text variable="container-title" />
+            </if>
+          </choose>
+        </else-if>
+      </choose>
+      <text macro="year" />
+      <choose>
+        <if type="legal_case" match="none">
+          <text macro="locator-range" />
+        </if>
+      </choose>
+    </group>
+  </layout>
+</citation>
+
+
+<bibliography hanging-indent="true">
+  <sort>
+    <key variable="author" />
+    <key variable="title" />
+  </sort>
+
+  <layout>
+    <!-- Author/Title -->
+    <choose>
+      <if type="entry-dictionary">
+        <choose>
+          <if variable="author">
+            <names variable="author">
+              <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol" />
+            </names>
+          </if>
+          <else>
+            <text macro="container-title" />
+          </else>
+        </choose>
+      </if>
+      <else-if type="legal_case  legislation">
+        <text macro="title" />
+      </else-if>
+      <else-if type="bill">
+        <text macro="title" />
+        <text macro="container-title" prefix=". " suffix=" Bill" />
+      </else-if>
+      <else-if type="webpage">
+        <choose>
+          <if variable="author">
+            <names variable="author">
+              <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol" />
+            </names>
+          </if>
+          <else-if variable="container-title">
+            <text macro="container-title" />
+          </else-if>
+          <else>
+            <text macro="title" />
+          </else>
+        </choose>
+      </else-if>
+      <else-if type="song" variable="composer">
+        <names variable="composer">
+          <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol" />
+        </names>
+      </else-if>
+      <else-if type="motion_picture" variable="medium">
```

Comment:
this only happens if you have a film _and_ the medium variable. Just making sure that's what you want.